controllers/storagecluster/clusterclaims.go

```diff
@@ -193,6 +203,26 @@ func (c *ClusterClaimCreator) getCephFsid() (string, error) {
 	return "", fmt.Errorf("failed to fetch ceph fsid from %q secret", RookCephMonSecretName)
 }
 
+func (c *ClusterClaimCreator) getIsDROptimized(serverVersion *version.Info) (string, error) {
```

Comment:
@sp98 Can you check if this is accurate?

Reply:
missed it. Checking now